src/server/license/server/api_server.go

```diff
@@ -35,11 +38,69 @@ func New(env Env) (lc.APIServer, error) {
 		env:     env,
 		license: licenseCollection(env.DB, env.Listener),
 	}
+	go s.envBootstrap(context.Background())
```

Comment:
Not 100% sure about this but I started a Slack discussion.

I think it would be best if we run this inline, not in the background, and return the error from envBootstrap here.  We do want to prevent the gRPC server from ever starting if envBootstrap fails; it would be weird to pass gRPC health checks for a while, and then when envBoostrap finally panics, shut down the server.  (We also don't want to handle any RPCs before auth is either activated, or we fail to activate it.  Seems like it would be hard to debug that that is happening.)